__tests__/app.test.js

```diff
@@ -441,4 +442,27 @@ describe("DELETE /api/comments/:comment_id", () => {
 })
 })
 
+describe("GET /api", () => {
+    it("200: Endpoint responds with a JSON describing all available endpoints for the API", () => {
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then((response) => {
+            console.log(response.body)
+            const returnedJson = response.body
```

Comment:
You could just check that the contents are the same as the json file you required

Reply:
I have now removed the tests other and just left the compare contents test.